src/main/java/io/kestra/plugin/notifications/teams/TeamsExecution.java

```diff
@@ -43,11 +42,12 @@
                 "    tasks:",
                 "      - id: teams",
                 "        type: io.kestra.plugin.notifications.teams.TeamsExecution",
+                "        url: \"https://TENANT-NAME.webhook.office.com/webhookb2/XXXXXXXXXX\"",
```

Comment:
```suggestion
                "        url: \"https://microsoft.webhook.office.com/webhookb2/XXXXXXXXXX\"",
```

Reply:
The lowest domain level is actually a Microsoft 365 tenant. Usually, the value is equal to the _company domain name_. Thus presenting the `microsoft` value there might IMHO be confusing a bit.

Reply:
Agreed but we want example to be as close as copy/paste working as possible, I think Microsoft 365 will be used a lot these days so I think it's good to have it as an example.